nvme_gw_cli.py

```diff
@@ -211,20 +216,74 @@ def create_namespace(self, args):
         except Exception as error:
             self.logger.error(f"Failed to add namespace: \n {error}")
 
-    @cli.cmd([argument("-n", "--subnqn", help="Subsystem NQN", required=True)])
-    def allow_any_hosts(self, args):
-        """Allows any host to access a subsystem."""
+    @cli.cmd([
+        argument("-n", "--subnqn", help="Subsystem NQN", required=True),
+        argument("-i", "--nsid", help="Namespace ID", type=int, required=True),
+    ])
+    def delete_namespace(self, args):
+        """Deletes a namespace from a previously created subsystem."""
+
+        try:
+            delete_req = pb2.ns_delete_req(subsystem_nqn=args.subnqn,
+                                           nsid=args.nsid)
+            ret = self.stub.nvmf_subsystem_remove_ns(delete_req)
+            self.logger.info(f"Deleted namespace {delete_req.nsid}: {ret}")
+        except Exception as error:
+            self.logger.error(f"Failed to remove namespace: \n {error}")
+
+    @cli.cmd([
+        argument("-n", "--subnqn", help="Subsystem NQN", required=True),
+        argument("-t", "--host", help="Host NQN", required=True),
+    ])
+    def add_host(self, args):
+        """Adds a host to a previously created subsystem."""
+
+        if args.host == "*":
+            allow_any_host = True
+
+        try:
+            create_req = pb2.subsystem_add_host_req(
+                subsystem_nqn=args.subnqn,
+                host_nqn=args.host,
+                allow_any_host=allow_any_host)
+            ret = self.stub.nvmf_subsystem_add_host(create_req)
+            if allow_any_host:
+                self.logger.info(
+                    f"Allow open host access to {args.subnqn}: {ret.status}")
```

Comment:
For consistency with other commands:
```suggestion
                    f"Allowed open host access to {args.subnqn}: {ret.status}")
```